### DISCUSSION TITLE
Welcome to RS-MET Discussions!

### DISCUSSION THREAD

**Discussion by RobinSchmidt** [RobinSchmidt/RS-MET#310] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by RobinSchmidt** [comment]
...well this was just the template text that github offered for the first discussion and i left it as is.

sooo - apparently github realized that the issue tracker tends to get "abused" for discussions a lot and has introduced this new feature to keep the discussions separated from the issues. very nice. it's also possible to convert issues to discussions, which i'll do soon where this is appropriate...

ah - there seem to be still some problems with that:
https://github.com/github/feedback/discussions/2924
...ok, so i'll wait until it's ready.

here's some general info about the new feature:

https://docs.github.com/en/discussions/managing-discussions-for-your-community/managing-discussions-in-your-repository

**Comment by denisreview** [comment]
👍💥

**Reply by RobinSchmidt** [reply]
ah - hi denis! nice to see you! looking forward to your daw project!

**Comment by denisreview** [comment]
Hi Robin! I have it here on Github already, just not made it public yet. So far looks a lot better here, comparing to Beanstalk. 

Btw, did you receive my latest ideas via email, regarding waveshaping, stretching, pitching, etc.?

**Reply by RobinSchmidt** [reply]
great to hear. yes - github seems to be the most major code hosting platform which just about everyone seems to use. it was actually acquired by microsoft some time ago which made me a bit wary about its future. but it seems like microsoft really wants to support open source software, which is a very welcome step for a giant commercial software vendor

yes, i still have to dig out the code for the time-stretch stuff. maybe we can continue the discussions now here on github with this new feature in place

**Comment by denisreview** [comment]
Ok, sure, there's plenty of time anyway, until the GUI is fully ready to integrate any internal DSP stuff. Just so you know, I'm planning to make the App first as a sampler with waveshaping, stretching, repitching & related stuff as a basic feature set.

**Comment by RobinSchmidt** [comment]
the feature for converting issues to discussions seems to work now. i'm now converting a bunch of issues into discussions. well done, github! "Discussions" is a very useful feature that was formerly missing.
